protected/models/Patient.php

```diff
@@ -2166,4 +2166,29 @@ public function checkdate($attribute,$params)
         }
 
     }
+
+    /**
+     * Find all patients with the same date of birth and similar-sounding names.
+     * @param $firstName string First name.
+     * @param $surname string Last name.
+     * @param $dob DateTime Date of Birth (DD/MM/YYYY).
```

Comment:
Yeah it probably is. I was wondering why the SQL worked if it wasn't a date object, yet apparently it does...